projects/laji/src/app/shared-modules/select/select/select.component.html

```diff
@@ -4,26 +4,36 @@
     <ng-container *ngIf="!useFilter; else multiWithFilter">
       <ng-container *ngIf="!open">
         <span class="item" *ngFor="let option of selectedOptions; trackBy: track">
-          <label class="lj-item selected" (click)="labelClick($event)">
-            <laji-checkbox [value]="true" (valueChange)="remove(option.id)"></laji-checkbox> {{ option.value}}
+          <label class="lj-item selected" [ngClass]="class.length > 0 ? class[0] : ''" (click)="labelClick($event)">
+            <ng-container *ngIf="checkboxType === 'basic' else checkboxSubCategoriesGlobal">
+              <laji-checkbox [value]="true" (valueChange)="remove(option.id, $event)"></laji-checkbox> {{ option.value}}
+            </ng-container>
+            <ng-template #checkboxSubCategoriesGlobal>
```

Comment:
There should not be need to use different checkbox handling since we now have checkboxType as an input to select component

Reply:
Now not different checkboxType 08e6e3f